### PR TITLE
Create 2024-10-08

### PR DESCRIPTION
**Pull request by bakeiro**
**Backend checklist**
- [ ] Formatted my code by running `ruff check --select I --fix . && ruff check . && ruff format .` in `app/backend`
- [ ] Added tests for any new code or added a regression test if fixing a bug
- [ ] All tests pass
- [ ] Run the backend locally and it works
- [ ] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

**Web frontend checklist**
- [ ] Formatted my code with `yarn format`
- [ ] There are no warnings from `yarn lint --fix`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: docs/meetings/dev/2024-10-08 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- We talked about how to coordinate interviews with new volunteers and how to schedule calls with them.
+- We talked about the profiles of new volunteers who want to contribute.
+- Discussed security measures for the database
+- Discussed the strong verification, which needs an interface that has not yet been created, Muriel volunteered to create it.
```
Comment (aapeliv) [inline docs/meetings/dev/2024-10-08]: Muriel -> Miuriel

File: docs/meetings/dev/2024-10-08 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+- We talked about how to coordinate interviews with new volunteers and how to schedule calls with them.
```
Comment (aapeliv) [inline docs/meetings/dev/2024-10-08]: Could you paste the following to the top of the page:

```
# Dev meeting

When: 8/10/24, 20:00 UTC.
Present: Aapeli, Darren, David, Ivan, Jesse, Miuriel, Nicole, Yannic

# Notes
```

File: docs/meetings/dev/2024-10-08 [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+- We talked about the profiles of new volunteers who want to contribute.
+- Discussed security measures for the database
+- Discussed the strong verification, which needs an interface that has not yet been created, Muriel volunteered to create it.
+- Aapeli: he has been very busy this week, so he has been able to progress it as he wanted to
+- Me: (David) Discussed a way to organize tasks and create tickets, so that the coming devs can easily understand what needs to be done, what milestone the task belongs to and possible mockups as figma mockups are often too complex to implement on the first interaction.
+- Ivan: He is progressing with the App, he finished some pending work.
+- Jesse: He has worked on the UX of the map and defined its behavior.
+- Darren: He is looking at a PR from Nicole about a migration of the Material UI components.
```
Comment (aapeliv) [inline docs/meetings/dev/2024-10-08]: Please also rename the file to include `.md` at the end!

**Review by aapeliv — CHANGES_REQUESTED** [review]
Sorry I am a perfectionist :(

Let me know if you want me to push a commit with these changes!

File: docs/meetings/dev/2024-10-08 (hunk 1 above)
Comment (bakeiro) [reply docs/meetings/dev/2024-10-08]: oh my god, thanks!!

File: docs/meetings/dev/2024-10-08 (hunk 2 above)
Comment (bakeiro) [reply docs/meetings/dev/2024-10-08]: sure!

File: docs/meetings/dev/2024-10-08 (hunk 3 above)
Comment (bakeiro) [reply docs/meetings/dev/2024-10-08]: 🫡

**Comment by bakeiro** [comment]
done all the changes @aapeliv!

**Review by aapeliv — APPROVED** [review]
Thank you for taking notes!!

**Pull request merged**